### PR TITLE
Hevc patches

### PR DESCRIPTION
**Pull request by feiwan1**

### CODE REVIEW

File: libavcodec/hevcdec.c [hunk 1]
```diff
@@ -594,15 +594,8 @@ static int hls_slice_header(HEVCContext *s)
 
     if (s->ps.sps != (HEVCSPS*)s->ps.sps_list[s->ps.pps->sps_id]->data) {
         const HEVCSPS *sps = (HEVCSPS*)s->ps.sps_list[s->ps.pps->sps_id]->data;
-        const HEVCSPS *last_sps = s->ps.sps;
         enum AVPixelFormat pix_fmt;
 
-        if (last_sps && IS_IRAP(s) && s->nal_unit_type != HEVC_NAL_CRA_NUT) {
-            if (sps->width != last_sps->width || sps->height != last_sps->height ||
-                sps->temporal_layer[sps->max_sub_layers - 1].max_dec_pic_buffering !=
-                last_sps->temporal_layer[last_sps->max_sub_layers - 1].max_dec_pic_buffering)
```
Comment (xhaihao) [inline libavcodec/hevcdec.c:603]: The commit log says resolution change, is the change of max_dec_pic_buffering resolution change ?

**Comment by xhaihao** [comment]
please replace `,` with `:` in the git shortlog `lavc/hevcdec, do not let missing ref frames invovle in dpb process `

File: libavcodec/hevcdec.c (hunk 1 above)
Comment (feiwan1) [reply libavcodec/hevcdec.c:603]: Refined the commit log. Thanks.

**Comment by feiwan1** [comment]
> please replace `,` with `:` in the git shortlog `lavc/hevcdec, do not let missing ref frames invovle in dpb process `

Fixed, thanks.

**Review by xhaihao — APPROVED** [review]
LGTM

**Comment by feiwan1** [comment]
Hi @bin1.wu@intel.com, could you help to check why this check fail? Not sure if the HW meets some issue:

libva: VA-API version 1.15.0
[2022-06-06 07:40:42.506998] DEBUG: slash: [AVHWDeviceContext @ 0x564dce821800] libva: User requested driver 'iHD'
[2022-06-06 07:40:42.507019] DEBUG: slash: [AVHWDeviceContext @ 0x564dce821800] libva: Trying to open /opt/media/install/lib/dri/iHD_drv_video.so
[2022-06-06 07:40:42.507239] DEBUG: slash: [AVHWDeviceContext @ 0x564dce821800] libva: Found init function __vaDriverInit_1_15
[2022-06-06 07:40:44.311116] DEBUG: slash: [AVHWDeviceContext @ 0x564dce821800] libva: /opt/media/install/lib/dri/iHD_drv_video.so init failed
[2022-06-06 07:40:44.311222] DEBUG: slash: [AVHWDeviceContext @ 0x564dce821800] libva: va_openDriver() returns 1
[2022-06-06 07:40:44.311250] DEBUG: slash: [AVHWDeviceContext @ 0x564dce821800] Failed to initialise VAAPI connection: 1 (operation failed).
[2022-06-06 07:40:44.314411] DEBUG: slash: Device creation failed: -5.

**Comment by feiwan1** [comment]
All test pass now, send the patch to community:
https://patchwork.ffmpeg.org/project/ffmpeg/list/?series=6761

**Pull request closed without merging**